src/Psy/Command/ReflectingCommand.php

```diff
@@ -65,10 +65,9 @@ protected function getTarget($valueName, $classOnly = false)
                 // @todo maybe do something interesting with these at some point?
                 if (array_key_exists($matches[1], $GLOBALS)) {
                     throw new RuntimeException('Unable to inspect a non-object');
-                } else {
-                    throw new RuntimeException('Unknown target: ' . $valueName);
                 }
 
+                throw new RuntimeException('Unknown target: ' . $valueName);
```

Comment:
Bit of a hack to get PHP-CS-Fixer to work. :trollface:

Reply:
else after `return`/`break`/`continue`/`throw` is always sad.

Reply:
:+1: